main.lua

```diff
@@ -41,13 +42,13 @@ for _, Folder in ipairs(cFile:GetFolderContents(WorldEditPath)) do repeat
 	dofolder(Path)
 until true end
 
-PLUGIN = nil
+
+
 
 
 function Initialize(a_Plugin)
-	PLUGIN = a_Plugin
-	PLUGIN:SetName(g_PluginInfo.Name)
-	PLUGIN:SetVersion(g_PluginInfo.Version)

```

Comment:
Exception added. But yes, this kind of code is ugly, especially all-caps variable names.